### DISCUSSION TITLE
Integrating filterSpecs and filterSpecsByRange

### DISCUSSION THREAD

**Discussion by jihoon-seo** [cloud-barista/cb-tumblebug#545] [General]
`filter-specs.sh`
```
		    "num_vCPU": 1, 
		    "mem_GiB": 2
```
주어진 조건에 대해 exact match 를 합니다.

`range-filter-specs.sh`
```
		    "num_vCPU": {
			    "min": 2,
			    "max": 4
		    }, 
		    "mem_GiB": {
			    "min": 4
		    },
		    "storage_GiB": {
			    "max": 400
		    }
```
각 필드에 대해
min 만 주거나
max 만 주거나
min 과 max 를 모두 줘서
범위로 검색할 수 있도록 합니다.

지금 살펴 보니
```
		    "num_vCPU": {
			    "min": 2,
			    "max": 2
```
이렇게 하면
exact match 를 할 수 있으니
filterSpecsByRange 하나로 합칠 수도 있기는 하겠네요.

**Comment by seokho-son** [comment]
@jihoon-seo 원래도 1단계 2단계로 차근차근 개발 중이었던 것으로 기억이 나네요. 단일화하면 좋을 것 같습니다.. ^^
